src/models/spaces/charts/chart-data-set.ts

```diff
@@ -228,7 +228,9 @@ export const ChartDataSetModel = types
     }
 
     function addDataPoint(a1: number, a2: number, label: string) {
-      self.dataPoints.push({ a1, a2, label });
+      if (!self.dataPoints.some(d => d.a1 === a1)) {
+        self.dataPoints.push({ a1, a2, label });
+      }
```

Comment:
This duplicate value filtering might be too aggressive.  It could cause slow-down or it could prevent some future functionality where we would want duplicate values for a given a1 (at present these data points represent values over time so inserting multiple values at a given a1 or t does not make sense).  The primary reason this is here is so that we do not duplicate the values at t = 0 (since we have a default set of values in the model and we add another set when the user presses start).  We could also add in a check to see if t ===0  (a1===0 at this point in the code) or stop the code from adding the duplicate values elsewhere.

Reply:
I agree that this seems a little heavy-handed for this small bug, and also seems the wrong place for the fix, to me. I feel like it would be better just to fix the issue that is causing it to put the data at time zero twice.

I forget the exact reason that we put the first data-point in before the model starts, but I think that that was explicitly desired. So we just shouldn't put the data there a second time when the model starts.

I think if you remove the entire listener `Events.addEventListener(Environment.EVENTS.START ...` in mouse-populations-model.ts`, it solves the problem. With a brief look, I haven't found any cases where we needed the data to be added on START.

Reply:
Agreed.  Fixed.